documentation/seed-node.md

```diff
@@ -0,0 +1,256 @@
+# Seed node commands
+
+Vulcanizedb can act as an index for Ethereum data stored on IPFS through the use of the `syncAndPublish` and
+`syncPublishScreenAndServe` commands. 
```

Comment:
Why not split it up into `syncAndPublish` and `screenAndServe`? Feels a bit odd to have commands partially overlapping feature wise.

Reply:
I like this idea although the functionality of `screenAndServe` would be limited to previously synced and published data if there is not a `syncAndPublish` process going on at the same time.

Reply:
Yeah, in the same way the application works when running `composeAndExecute` without `headerSync`? :blush:

Reply:
Yeah that is a good analogy